application/backend/app/services/datumaro_converter.py

```diff
@@ -40,6 +39,7 @@ class ClassificationSample(Sample):
     image_info: ImageInfo = image_info_field()
     label: int = label_field(dtype=pl.Int32(), is_list=False)
     confidence: float | None = score_field(dtype=pl.Float32())
+    subset: Subset = subset_field()
```

Comment:
```suggestion
    subset | None: Subset = subset_field()
```
We can use the `None` value to mean "unassigned". Note that these Sample classes will be later reused for dataset import/export too, where it makes sense to have samples without a subset.

Reply:
@AlbertvanHouten

When defining the field as `subset: Subset | None = subset_field()`, an error occurs during sample retrieval with `dataset[0]`. Should the `Subset` be extended with an `UNASSIGNED` type on the Datumaro side, or are there other proposals?

```
target_type = datumaro.experimental.fields.Subset | None

    def from_polars(self, name: str, row_index: int, df: pl.DataFrame, target_type: type[T]) -> T:
        """Reconstruct subset value from Polars data.
    
        If target_type is an Enum, converts string back to enum value.
        Otherwise, returns the string value directly.
        """
        value = df[name][row_index]
    
        if value is None:
            return None  # type: ignore
    
>       if issubclass(target_type, Enum):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       TypeError: issubclass() arg 1 must be a class

application/backend/.venv/lib/python3.13/site-packages/datumaro/experimental/fields.py:1174: TypeError
```

Reply:
Upgraded `datumaro` and using `UNASSIGNED` in conversions.